Project.toml

```
@@ -1,11 +1,11 @@
 name = "NCBIBlast"
 uuid = "dcaf3faf-48b5-45a7-abb1-fcfd75344536"
 authors = ["Kevin Bonham <kevin@bonham.ch> and contributors"]
-version = "0.1.0"
+version = "0.2.0"
 
 [deps]
-CondaPkg = "992eb4ea-22a4-4c89-a5bb-47a3300528ab"
+Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-CondaPkg = "0.2.23"
+Artifacts = "1.11.0"
```

```suggestion
Artifacts = "1"
```

I think as-written,  it pins to julia v1.11 based on test failure

---

I think Artifacts lived inside Pkg in 1.10, updated to use Pkg instead.